### PR TITLE
Fix: Replace map.initialViewpoint with mapView.setViewpoint

### PR DESCRIPTION
**Pull request by yo1995**
This PR replaces the `map.initialViewpoint` with `mapView.setViewpoint` to follow the best practice of 
> *`mapView` handles display; `map` handles content*

Please see more discussion at `common-samples/issues/2479`

Some additional changes are made to clean up comment and to standardize coding style. However, if it involves too many (irrelevant) line-of-change in a file to fix styling, then it is left untouched to avoid making it hard to review.

---

Not changed:
- BookmarksViewController: the `initialViewpoint` save with the map, which makes sense for bookmark
- Create and save map: when saving the map, it also saves the initial viewpoint of the map
- LineOfSightGeoElementViewController: the initial view point is set on a scene.

### CODE REVIEW

**Review by vquach2404 — APPROVED** [review]

File: arcgis-ios-sdk-samples/Layers/Identify KML features/IdentifyKMLFeaturesViewController.swift [hunk 1]
```diff
@@ -36,10 +36,9 @@ class IdentifyKMLFeaturesViewController: UIViewController {
         let map = AGSMap(basemapStyle: .arcGISDarkGrayBase)
         map.operationalLayers.add(forecastLayer)
         
-        let center = AGSPoint(x: -48_885, y: 1_718_235, spatialReference: AGSSpatialReference(wkid: 5070))
-        map.initialViewpoint = AGSViewpoint(center: center, scale: 50_000_000)
-        
         mapView.map = map
+        let center = AGSPoint(x: -48_885, y: 1_718_235, spatialReference: AGSSpatialReference(wkid: 5070))
```
Comment (philium) [inline arcgis-ios-sdk-samples/Layers/Identify KML features/IdentifyKMLFeaturesViewController.swift:40]: Thank you for moving this variable. In some cases, due to the creation of the viewpoint moving, the variable for the point used as the center of the viewpoint is separated from creating and setting the viewpoint itself.

File: arcgis-ios-sdk-samples/Layers/Manage sublayers/ManageSublayersViewController.swift [hunk 2]
```diff
@@ -41,12 +41,12 @@ class ManageSublayersViewController: UIViewController, MapImageSublayersViewCont
         //initial viewpoint
         let envelope = AGSEnvelope(xMin: -13834661.666904, yMin: 331181.323482, xMax: -8255704.998713, yMax: 9118038.075882, spatialReference: .webMercator())
         
-        //set initial viewpoint on map
-        map.initialViewpoint = AGSViewpoint(targetExtent: envelope)
-        
         //assign map to map view
         self.mapView.map = map
         
+        //set initial viewpoint on map
```
Comment (philium) [inline arcgis-ios-sdk-samples/Layers/Manage sublayers/ManageSublayersViewController.swift:47]:
```suggestion
        //set the viewpoint on map view
```

File: arcgis-ios-sdk-samples/Maps/Map rotation/MapRotationViewController.swift [hunk 3]
```diff
@@ -26,24 +26,24 @@ class MapRotationViewController: UIViewController {
...
+        // Update the slider value when the user rotates by pinching.
+        mapView.viewpointChangedHandler = { [weak self] in
             DispatchQueue.main.async {
                 self?.mapViewpointDidChange()
             }
         }
 
-        //initial viewpoint
-        self.map.initialViewpoint = AGSViewpoint(targetExtent: AGSEnvelope(xMin: -13044000, yMin: 3855000, xMax: -13040000, yMax: 3858000, spatialReference: .webMercator()))
+        // Set initial viewpoint.
```
Comment (philium) [inline arcgis-ios-sdk-samples/Maps/Map rotation/MapRotationViewController.swift:45]:
```suggestion
        // Set the map view's viewpoint.
```

File: arcgis-ios-sdk-samples/Maps/Set initial map area/SetInitialMapAreaViewController.swift [hunk 4]
```diff
@@ -17,22 +17,21 @@ import ArcGIS
...
-        //initialize the map with imagery basemap
-        self.map = AGSMap(basemapStyle: .arcGISImageryStandard)
+        // Initialize the map with imagery basemap.
+        let map = AGSMap(basemapStyle: .arcGISImageryStandard)
+
+        // Assign the map to the map view.
+        mapView.map = map
         
-        //set initial map area
+        // Set initial map area.
```
Comment (philium) [inline arcgis-ios-sdk-samples/Maps/Set initial map area/SetInitialMapAreaViewController.swift:33]:
```suggestion
        // Set the map view's viewpoint.
```

File: arcgis-ios-sdk-samples/Maps/Set initial map location/SetInitialMapLocationViewController.swift [hunk 5]
```diff
@@ -17,20 +17,18 @@ import ArcGIS
...
     
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        //add the source code button item to the right of navigation bar
+        // Add the source code button item to the right of navigation bar.
         (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["SetInitialMapLocationViewController"]
         
-        //initialize map with `imagery with labels` basemap and an initial location
+        // Initialize map with imagery basemap and an initial location.
```
Comment (philium) [inline arcgis-ios-sdk-samples/Maps/Set initial map location/SetInitialMapLocationViewController.swift:27]: This comment is no longer accurate as the map is no longer configured with a (custom) initial location (viewpoint).

File: arcgis-ios-sdk-samples/Features/Feature layer (geodatabase)/FeatureLayerGDBViewController.swift [hunk 6]
```diff
@@ -29,24 +29,23 @@ class FeatureLayerGDBViewController: UIViewController {
...
+            guard let self = self else { return }
             if let error = error {
-                self?.presentAlert(error: error)
+                self.presentAlert(error: error)
             } else {
-                let featureTable = self!.geodatabase.geodatabaseFeatureTable(withName: "Trailheads")!
+                let featureTable = self.geodatabase.geodatabaseFeatureTable(withName: "Trailheads")!
                 let featureLayer = AGSFeatureLayer(featureTable: featureTable)
-                self?.mapView.map?.operationalLayers.add(featureLayer)
+                self.mapView.map?.operationalLayers.add(featureLayer)
```
[the remark this answers is not in the document]
Comment (yo1995) [reply arcgis-ios-sdk-samples/Features/Feature layer (geodatabase)/FeatureLayerGDBViewController.swift:47]: Yes 🤦  Initially I thought it is OK to do a few cleanups along the way, but as the PR grows larger, I regretted... thus didn't make additional changes.

I'll revert these extra changes.

**Review by philium — APPROVED** [review]

**Pull request merged**